BNote/src/presentation/modules/repertoireview.php

```diff
@@ -337,11 +338,6 @@ private function songFiles() {
 	public function addSongFile() {
 		$songId = $_GET["id"];
 		$fullpath = $_POST["file"];
-		// check file's existence

```

Comment:
Actually I think we do need the file existence check. This was reported as a bug...